app/controllers/gems_controller.rb

```diff
@@ -1,4 +1,6 @@
 class GemsController < ApplicationController
   def index
+    @gems = Gems.gems(current_user.username)
```

Comment:
This is going to cause you all sorts of bugs.  Your current user is a GitHub user but the library is expecting a RubyGems.org user. They might be the same, but they might not be.